plugins/Marketplace/Consumer.php

```diff
@@ -22,6 +24,11 @@ class Consumer
     private $consumer = false;
     private $isValid = null;
 
+    /**
+     * @var array
+     */
+    private $consumerLicenseStatusPluginWise = null;
```

Comment:
I'm not sure I understand the full variable name, can we do something like `licenseStatusesForPlugins` or `pluginLicenseStatuses`?

Reply:
If we want to keep it future proofed and allow for other license related information as mentioned in the other comment, perhaps the variable can be named `pluginLicenseInfo`.
It's in the consumer class, so the consumer prefix is superfluous.